.circleci/config.yml

```diff
@@ -6,7 +6,7 @@ jobs:
   build:
     working_directory: ~/project
     docker:
-      - image: cimg/php:8.1-browsers
+      - image: cimg/php:8.1.18-browsers
```

Comment:
This can be reverted once the phpcs error is fixed, https://github.com/php/php-src/issues/11972

Reply:
@raylwalters @becw based on the linked PHP issue, should we have pinned to 8.1.17?

Reply:
I thought we were having issues with 8.1.19. I have PNET pinned to 8.1.18 and haven't been seeing the errors.